toplevel/examples/lwt_toplevel/dune

```diff
@@ -39,7 +39,10 @@
   (:standard -linkall))
  (modes byte js)
  (js_of_ocaml
-  (link_flags (:standard))
+  (link_flags
+   :standard
+   ; The source map would be too large on 32-bit architectures otherwise
+   --no-source-map)
```

Comment:
Wouldn't it be enough to disable the inline sourcemap ?
```
(link_flags (:standard --source-map \ --source-map-inline)) 
```

Or maybe just disable sourcemap on 32bit arch only

Reply:
Yes, this is enough.

I'm not sure how to disable sourcemap on 32bit arch only.

Reply:
> I'm not sure how to disable sourcemap on 32bit arch only.

Emit flags in a file and use `(:include filename)` in the dune stanza.

Reply:
Do you think it is worth it?

Reply:
Looking more at it. Having sourcemap enabled here was probably a mistake in the first place. We commit the generated file in the wikidoc branch. Better to not  bloat it with sourcemap.